_data/students.yml

```diff
@@ -11,6 +11,13 @@
 #   lat: Student's Latitude
 #   lng: Student's Longitude
 
+ - name: Shivam Rai
+   country:India
+   github: ShivamRai2003
+   image:https://avatars3.githubusercontent.com/u/34955127?s=460&v=4
```

Comment:
space after  image:
and you are supposed to write the name of the image that you have uploaded